WoWsShipBuilder.Core/DataContainers/Aircraft/CvAircraftDataContainer.cs

```diff
@@ -7,6 +7,8 @@
 using WoWsShipBuilder.DataElements.DataElementAttributes;
 using WoWsShipBuilder.DataElements.DataElements;
 using WoWsShipBuilder.DataStructures;
+using WoWsShipBuilder.DataStructures.Aircraft;
+using WoWsShipBuilder.DataStructures.Ship;
 
 namespace WoWsShipBuilder.Core.DataContainers
```

Comment:
There is a dangling //TODO that also gives a warning in at row 103, just after PlaneConsumables

Reply:
That Todo is not from me but from itto